test/test/nonjava/common_non_java.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _COMMON_NONE_JAVA_TEST_H
+#define _COMMON_NONE_JAVA_TEST_H
+
+#include <dlfcn.h>
+#include <jni.h>
+#include <unistd.h>
+#include <iostream>
+#include "asprof.h"
+
+typedef jint (*CreateJvm)(JavaVM **, void **, void *);
+
+class CommonNonJava {
```

Comment:
In what sense it is "common"?
Having 3 source files for a couple of tests seems an overkill. Can it be just one .cpp file?